recipe/build.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./modules/oniguruma
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./config
```

Comment:
@jakirkham, see here

Reply:
Thanks. Was more trying to get a sense of what these files are doing, but I'm fine just trusting you 🙂

Reply:
Oh. These files `config.sub`, `config.guess` are vendored in every autotools project and they are outdated in most projects. These outdated files don't recognise the new platform `arm64-apple-darwin` and fail. (anything before July 2020 is outdated)

Reply:
Ah ok. That makes sense. Thanks for clarifying.

Should we patch our autotools packages to just use/include these?

Reply:
they are vendored in source tarballs of autotools-based packages for eg: `jq` source tarball has these.

Reply:
Ah got it. Would running `autoreconf` help?

Reply:
See https://github.com/conda-forge/bison-feedstock/pull/27#discussion_r523277854

Reply:
Sorry 🤦‍♂️ Had forgotten we already discussed this